nautobot/ipam/templates/ipam/ipaddress_assign.html

```diff
@@ -39,10 +46,21 @@ <h3>Assign an IP Address</h3>
     </form>
     {% if table %}
         <div class="row">
-            <div class="col-md-12" style="margin-top: 20px">
-                <h3>Search Results</h3>
-                {% include 'utilities/obj_table.html' with table_template='panel_table.html' %}
-            </div>
+            <form action="{% querystring request %}" method="post" class="form form-horizontal">
+                {% csrf_token %}
+                <div class="col-md-12" style="margin-top: 20px">
+                    <h3>Search Results</h3>
+                    {% include 'utilities/obj_table.html' with table_template='panel_table.html' disable_pagination=True %}
+                    <div class="pull-left noprint">
+                        {% if perms.dcim.change_interface %}
+                            <button type="submit" name="_edit" formaction="#" class="btn btn-primary btn-sm">
+                                <span class="mdi mdi-clipboard-tick" aria-hidden="true"></span> Assign Selected IPAddresses to Interface
+                            </button>
+                        {% endif %}
+                    </div>
+                </div>
+            </form>
+            {% include 'inc/paginator.html' with paginator=table.paginator page=table.page %}
```

Comment:
Why do we have/need a paginator if `disable_pagination` is True?

Reply:
We had to do `{% include 'utilities/obj_table.html' with table_template='panel_table.html' disable_pagination=True %}` and then render the paginator ourselves later in the file. Otherwise we ended up with nested `<form>` elements which caused the paginator to submit the wrong form.